pkgs/kde/frameworks/breeze-icons/default.nix

```diff
@@ -11,6 +12,11 @@ mkKdeDerivation {
     libxml2
   ];
 
+  # Fixes missing icons in non-qt desktop envirunments
```

Comment:
Maybe `This package contains an SVG icon theme and an API forcing its use` would be more accurate?

Reply:
amended it